components/ingest-service/pipeline/processor/transmogrify_run.go

```diff
@@ -80,6 +80,9 @@ func ChefRunTransmogrify(in <-chan message.ChefRun, out chan<- message.ChefRun,
 				continue
 			}
 
+			// Needed for the nodemanager because the platform field is combined with the version.
+			msg.Platform = ccr.Platform()
```

Comment:
This is the only place we have access to the raw platform. So, we set the pipeline msg platform here.